beyond-essence/inventory/toggle-switches-updated.md

```diff
@@ -27,7 +27,7 @@ Customers will be able to view and interact with the merchant's website. This is
 * **Allow visitors to make purchases on your website**: When the merchant activates this toggle, a cart will be displayed at the top of the website and each product will have an "Add to cart" button, allowing visitors to order the merchant's products.
 The merchant needs to make sure they are ready to sell (and not only showcase) their products on their website before activating the toggle.
 
-💡 _Note: This is a temporary solution and might be handled differently in the future._
+💡 _Note: This is an inherited and temporary solution for this use case. The current implementation needs to be reviewed and updated in the future._
```

Comment:
The second one (allow visitors to make purchases on your website) wasn't inherited - I designed that (under duress). It's more that I had to do it that way because we had already inherited the first use case in our software. And there was no way to do it differently. So I guess you could just take out "for this use case".

Reply:
@sarahwr So do you want it to say "This is an inherited and temporary solution. The current implementation needs to be reviewed and updated in the future." or "This is a temporary solution. The current implementation needs to be reviewed and updated in the future." ?
Are you ok with everything else I had added, e.g. the new part "Switch between different modes" that you requested with your last review?